.vsts-ci.yml

```diff
@@ -126,7 +126,7 @@ extends:
           parameters:
             agentOs: Linux
             jobName: Build_CentOS_8_Stream_Debug_x64
-            container: centosStream8
+            container: centosStream9
             buildConfiguration: Debug
```

Comment:
Mismatch in CentOS version between `jobName` and `container`. Suggest changing `jobName` to "Build_CentOS_9_Stream_Debug_x64".

Reply:
Fixed. Thanks!